cmd/juju/commands/bootstrap.go

```diff
@@ -78,10 +89,13 @@ use in all models for this controller. The full binary version is accepted
 used. Otherwise, by default, the version used is that of the client.
 
 Examples:
-    juju bootstrap mycontroller google
-    juju bootstrap --config=~/config-rs.yaml mycontroller rackspace
-    juju bootstrap --config agent-version=1.25.3 mycontroller aws
-    juju bootstrap --config bootstrap-timeout=1200 mycontroller azure
+    juju bootstrap
```

Comment:
maybe add a "# interactive bootstrap" afterwards?
and similar comments for "--clouds" and "--regions aws"?